src/System.Net.Http/src/System/Net/Http/Managed/AuthenticationHelper.Digest.cs

```diff
@@ -126,15 +126,8 @@ internal partial class AuthenticationHelper
             if (realm != string.Empty)
                 sb.AppendKeyValue(Realm, realm);
 
-            // If nonce is same as previous request, update nonce count.
-            if (nonce == digestResponse.Nonce)
-            {
-                digestResponse.NonceCount++;
```

Comment:
Is NonceCount ever set to anything other than 1?  What is its purpose and why is it still needed?

(Also, is this just a holdover from some other change?  I don't understand its relationship to "Use Span in parsing string in Digest auth")

Reply:
The digestResponse.Nonce is not required anymore, since we removed Caching. So just clubbed it with this change. The NonceCount is always going to be 1 for every new DigestResponse, I can make it as a const..